ludwig/models/trainer.py

```diff
@@ -1098,6 +1098,30 @@ def append_metrics(self, dataset_name, results, metrics_log, tables, progress_tr
 
         return metrics_log, tables
 
+    def calibration(self, dataset, dataset_name: str, save_path: str):
```

Comment:
This function look suspiciously similar to `Calibrator.calibration`. Do we need both? Can't we use only one?

Reply:
Good catch.  I moved this from Trainer to Calibrator, hadn't deleted it from Trainer.